tiledb/bioimg/converters/base.py

```diff
@@ -147,6 +157,14 @@ def to_tiledb(
         :param tiles: A mapping from dimension name (one of 'T', 'C', 'Z', 'Y', 'X') to
             the (maximum) tile for this dimension.
         :param preserve_axes: If true, preserve the axes order of the original image.
+        :param namespace: The namespace to create the group in.
+            If not provided, the current logged-in user will be used.
+        :param parent_uri: The parent URI to add the group to, if desired.
+        :param storage_uri: The backend URI where the group will be stored.
+            If not provided, uses the namespace's default storage path for groups.
+        :param credentials_name: The name of the storage credential to use for
+            creating the group. If not provided, uses the namespace's default
+            credential for groups.
```

Comment:
Not very keen on having the `to_tiledb` signature being polluted by a bunch of (optional, rarely used) pass-through parameters relevant only to tiledb.cloud registration. I'd consolidate all four to a single `register_kwargs` dict parameter.